lib/rubocop/cop/style/preferred_hash_methods.rb

```diff
@@ -3,17 +3,50 @@
 module RuboCop
   module Cop
     module Style
-      # This cop checks for uses of methods Hash#has_key? and Hash#has_value?
-      # Prefer to use Hash#key? and Hash#value? instead
+      # This cop (by default) checks for uses of methods Hash#has_key? and
+      # Hash#has_value? where it preferences Hash#key? and Hash#value?
+      # It is configurable to enforce the opposite, `verbose`, methods
+      # also.
+      #
+      # @example
+      #
+      #  # EnforcedStyle: short (default)
+      #
+      #  # bad
+      #  Hash#has_key?
+      #  Hash#has_value?
+      #
+      #  # good
+      #  Hash#key?
+      #  Hash#value?
+      #
+      # @example
+      #
+      #  # EnforcedStyle: verbose
+      #
+      #  # bad
+      #  Hash#key?
+      #  Hash#value?
+      #
+      #  # good
+      #  Hash#has_key?
+      #  Hash#has_value?
       class PreferredHashMethods < Cop
+        include ConfigurableEnforcedStyle
+
         MSG = 'Use `Hash#%s` instead of `Hash#%s`.'.freeze
 
-        PREFERRED_METHODS = [:has_key?, :has_value?].freeze
+        OFFENDING_SELECTORS = {
+          style: {
```

Comment:
Only added that to fit in with existing coding style; see https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/lambda.rb#L66

Happy to remove :)